### PR TITLE
Finished ImagesRepository.

### PR DESCRIPTION
**Pull request by teamagam3**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/102" target="_blank">ReviewNinja

File: domain/src/main/java/com/teamagam/gimelgimel/domain/images/repository/ImagesRepository.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.teamagam.gimelgimel.domain.images.repository;
+
+import com.teamagam.gimelgimel.domain.messages.entity.MessageImage;
+
+public interface ImagesRepository {
+    void uploadImage(MessageImage message, byte[] imageBytes);
```
Comment (teamagam1) [inline domain/src/main/java/com/teamagam/gimelgimel/domain/images/repository/ImagesRepository.java]: you should use URI instead

**Comment by teamagam1** [comment]
move restAPI back where it belongs

File: data/src/main/java/com/teamagam/gimelgimel/data/images/ImagesDataRepository.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        MultipartBody.Part body = createMultipartBody(imageBytes);
+
+        mApi.sendImage(messageData, body);
+    }
+
+    private MultipartBody.Part createMultipartBody(byte[] imageBytes) {
+        MediaType mimeType = MediaType.parse(Constants.IMAGE_MIME_TYPE);
+        RequestBody requestFile =
+                RequestBody.create(mimeType, imageBytes, 0, imageBytes.length);
+        String randomFileName = UUID.randomUUID().toString();
```
Comment (teamagam1) [inline data/src/main/java/com/teamagam/gimelgimel/data/images/ImagesDataRepository.java]: no need for this UUID

**Comment by teamagam1** [comment]
Im not sure who needs to do the compressing. I think it should be done by the domain,

**Pull request merged**